frontend/src/pages/modelRegistry/screens/components/DeployRegisteredModelModal.tsx

```diff
@@ -51,16 +55,35 @@ const DeployRegisteredModelModal: React.FC<DeployRegisteredModelModalProps> = ({
     error: deployInfoError,
   } = useRegisteredModelDeployInfo(modelVersion);
 
+  const handleSubmit = React.useCallback(async () => {
+    if (!modelVersion.registeredModelId) {
+      return;
+    }
+
+    try {
+      await Promise.all([
+        modelRegistryApi.api.patchModelVersion({}, { state: ModelState.LIVE }, modelVersion.id),
+        modelRegistryApi.api.patchRegisteredModel(
+          {},
+          { state: ModelState.LIVE },
+          modelVersion.registeredModelId,
+        ),
+      ]);
```

Comment:
You still have direct patch calls here where I think you need to be using `bumpBothTimestamps` instead to make sure the _lastModified property workaround gets applied